### PR TITLE
regex

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
code pr https://github.com/vesoft-inc/nebula-graph/pull/414/files

### CODE REVIEW

**Review by jievince — DISMISSED** [review]
Good Job!

File: docs-2.0/3.ngql-guide/5.operators/7.string.md [hunk 1]
```diff
@@ -138,3 +139,24 @@ nebula> YIELD 'apple' ENDS WITH 'b';
...
+
+* Regular expressions
+
+Nebula Graph supports filtering by using regular expressions. The regular expression syntax is inherited from `st::regex`. You can match on regular expressions by using `=~ 'regexp'`. For example:
+
+```ngql
+nebula> FETCH PROP ON player WHERE player.name =~ '_?[A-Za-z]*[1-9]*' YIELD player.name;
+```
+
+The preceding statement returns players with name `_Var123` or `VAR`.
```
Comment (Shylock-Hg) [inline docs-2.0/3.ngql-guide/5.operators/7.string.md:151]: The explanation of preceding statement seems mistake.

**Review by jievince — DISMISSED** [review]

File: docs-2.0/3.ngql-guide/5.operators/7.string.md [hunk 2]
```diff
@@ -138,3 +139,13 @@ nebula> YIELD 'apple' ENDS WITH 'b';
 | false               |
 +---------------------+
 ```
+
+* Regular expressions
+
+Nebula Graph supports filtering by using regular expressions. The regular expression syntax is inherited from `st::regex`. You can match on regular expressions by using `=~ 'regexp'`. For example:
+
+```ngql
+nebula> FETCH PROP ON player WHERE player.name =~ 'Tony.*' YIELD player.name;
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/5.operators/7.string.md:148]: ^[0-9]+(.[0-9]{2})?$

Comment (whitewum) [inline docs-2.0/3.ngql-guide/5.operators/7.string.md:148]: ^[0-9]+(.[0-9]{2})?$

File: docs-2.0/3.ngql-guide/5.operators/7.string.md [hunk 3]
```diff
@@ -138,3 +139,13 @@ nebula> YIELD 'apple' ENDS WITH 'b';
...
+```
+
+The preceding statement returns players whose name starts with 'Tony'.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/5.operators/7.string.md:151]: players whose name starts with 'Tony'. ==> the players whose names start with 'Tony'.

**Review by jievince — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**